packages/validators/src/db/index.ts

```diff
@@ -43,3 +44,8 @@ export const ProfileSchema = z.object({
   id: z.string().min(0).max(255),
   username: username,
 });
+
+export const RestaurantSchema = z.object({
+  restaurantId: z.string().min(0).max(50),
+  name: z.string().min(0).max(255),
```

Comment:
```suggestion
  restaurantName: z.string().min(0).max(255),
```
You should use the same min and max lengths from here for the rest of the validators

Reply:
done